draft-iab-covid19-workshop.md

```diff
@@ -273,25 +273,27 @@ impact on networking. With large numbers of people working from home
 or otherwise depending on the network for their daily lives, network
 traffic volume has surged. Internet service providers and operators have
 reported a 20% traffic growth or more in a matter of weeks. Traffic at
-Internet Exchange Points (IXPs) is similarly on the rise. Most forms
+Internet Exchange Points (IXPs) is similarly on the rise, as were attacks targeting remote working infrastructure and platforms. Most forms
```

Comment:
I don't think just adding this half-sentence here helps. The point here is about increase in traffic volume (without characterising the traffic at this point). It would probably be much better to add a separate paragraph about attack traffic rather then trying to "sneak" it in at places where it doesn't belong.